frontend/src/components/TabSwitcher/TabSwitcher.tsx

```diff
@@ -36,6 +45,26 @@ const TabSwitcher = ({ children }: TabSwitcherProps) => {
     });
   };
 
+  const generateCols = () => {
+    const sharedCols = 'Date,Daily Total';
+
+    if (currentTab === 'Rider Data') {
+      return `${sharedCols},Daily Ride Count,Day No Shows,Day Cancels, \
+      Night Ride Count, Night No Shows, Night Cancels`
+    } else {
+      const driverNamesReducer = (acc: string, curr: Driver) => 
+      `${acc},${curr.firstName} ${curr.lastName.substring(0, 1)}.`
+
+      const driverShortNames = drivers
+      .sort((a, b) => (
+        `${a.firstName} ${a.lastName}` < `${b.firstName} ${b.lastName}` ? -1 : 1
+      ))
+      .reduce(driverNamesReducer, sharedCols);
```

Comment:
we actually want all ride and driver data to be exported, so you can combine all the columns